product/module.go

```diff
@@ -76,7 +76,7 @@ commerce: {
 		fakeservice: {
 			enabled: bool | *false
 			currency: *"€" | !=""
-			jsonTestDataFolder: *"testdata/products" | !=""
+			jsonTestDataFolder?: !=""
```

Comment:
now we loose the default value. I'm not sure either what would be the most comfortable way, but how about something like this
```
if enabled {
  jsonTestDataFolder: string | *"testdata/products"
}
```